source/Ship.cpp

```diff
@@ -3132,18 +3156,26 @@ void Ship::UnloadBays()
 			bay.ship->SetPlanet(landingPlanet);
 			bay.ship->UnmarkForRemoval();
 			bay.ship.reset();
+			bay.free = true;
```

Comment:
When a fighter is unloaded from a bay, doesn't its reserved bay remain the bay that it was unloaded from? In that case, shouldn't the bay not be considered free?